lib/player.rb

```diff
@@ -46,5 +49,65 @@ def highest_word_score
       return Scrabble.score(highest_scoring_word)
     end
 
+    def draw_tiles(tile_bag)
+      raise ArgumentError if (tile_bag.length > 7)
+      num_tiles = 7 - tile_bag.length
+      tile_bag.push(@tile_bag.draw_tiles(num_tiles))
+      return tile_bag.flatten!
+    end
   end
+
+ class Player
+   attr_accessor :name, :score, :plays, :score_array, :tiles_array, :tiles
+
+
+   def initialize(name)
+     @name = name
+     @score = 0
+     @plays = []
+     @score_array = []
+     @tiles= []
+     @tile_bag = TileBag.new
+   end
+
+
+   def play(word)
+     if won?
```

Comment:
Now that we know about the ternary operator (`condition ? true_case : false_case`) how might we simplify this if statement?